PC/config.c

```diff
@@ -122,10 +124,12 @@ struct _inittab _PyImport_Inittab[] = {
     {"itertools", PyInit_itertools},
     {"_collections", PyInit__collections},
     {"_symtable", PyInit__symtable},
+#if !defined(MS_WINDOWS_NON_DESKTOP) || defined(MS_WINDOWS_GAMES)
```

Comment:
just one is hard without more defined, since we have the following distinctions:
- windows desktop only
- windows but not games
- windows and games but not other non desktop

For the third case we do not have a separate macro